Packages/com.chisel.components/Chisel/Components/API.public/Components/Containers/ChiselGeneratedObjects.cs

```diff
@@ -597,6 +597,7 @@ public bool HasLightmapUVs
                 }
             }
 
+
```

Comment:
Nitpicking here, but when you do make a PR, try not to include files where the only change is whitespace. It just makes going through the history of a file more confusing

Reply:
Will clean, thanks for the catch